### PR TITLE
Initial GitHub Action workflow for testing docker-java-api against various JDK versions

### PR DESCRIPTION
**Pull request by geertjanw**

### CODE REVIEW

**Comment by amihaiemil** [comment]
@geertjanw Do you think this is needed? What's the added value compared to Travis and Rultor? 

At the moment Travis is configured only for JDK 8 I think, but we can configure it to run on multiple JDKs.
We already have a library which does that: https://github.com/decorators-squad/eo-yaml/blob/master/.travis.yml and it looks like this: https://travis-ci.org/github/decorators-squad/eo-yaml

**Comment by geertjanw** [comment]
Well, GitHub Actions are integrated directly into GitHub, that's the advantage, plus easy to configure, and here you see what you'll be able to see in your own repo, automatically, whenever you push new code to the repo:

https://github.com/geertjanw/docker-java-api/actions/runs/332251353

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    name: Test JDK ${{ matrix.java }}, ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Test with Maven
+      run: mvn test -B --file pom.xml
```
Comment (amihaiemil) [inline .github/workflows/maven.yml:23]: @geertjanw I think the command here should be the same as in other cases: ``mvn clean install -Pcheckstyle,itcases``

Can you modify it please?

**Review by amihaiemil — CHANGES_REQUESTED** [review]
@geertjanw ok, let's go with it. I have just one comment regarding the test command.

File: .github/workflows/maven.yml (hunk 1 above)
Comment (geertjanw) [reply .github/workflows/maven.yml:23]: But then we're building and not testing, i.e., not running tests only, which is what this GitHub Action is about. We can change it to do that, but that's something different.

Comment (amihaiemil) [reply .github/workflows/maven.yml:23]: @geertjanw if ``mvn test`` runs both unit tests (surefire plugin) and integration tests (failsafe plugin), then I am ok with it. 

If not, please specify the full command : D

Comment (geertjanw) [reply .github/workflows/maven.yml:23]: Yup, it does. :-)

**Comment by amihaiemil** [comment]
@rultor merge it please

**Comment by amihaiemil** [comment]
@geertjanw thanks!

**Comment by amihaiemil** [comment]
@geertjanw By the way, I'm not sure if or what needs to be done further. Will your file automatically set and trigger everyting up?

**Pull request merged**